project/Build.scala

```diff
@@ -9,36 +9,15 @@ import com.typesafe.sbt.SbtGit._
 
 object Build extends Build {
 
-  val baseVersion = "4.0.0"
+  val baseVersion = "5.0.0"
```

Comment:
we haven't released 4.0.0 yet. just 4.0.0-RC1, so we could just call this RC2 if you wanted